src/util/hass-util.html

```diff
@@ -69,6 +71,16 @@
   }
 };
 
+// If root has lastChild, set 'isVisible' attribute of that child to false.
+window.hassUtil.resetDynamicContent = function (root) {
```

Comment:
Removing the more-info tag is not good enough - the stream continues to run. The src must be changed to non-stream to stop it.

I'll inline resetDynamicContent into more-info-content.html